samples/event-hubs-application-insights-logging/EventLevelExtensions.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.ApplicationInsights.DataContracts;
+using System.Diagnostics.Tracing;
+
+namespace AzureEventSourceListenerEventHubsLogging
+{
+    /// <summary>
+    /// Severity level extensions for <see cref="EventLevel"/>.
+    /// </summary>
+    static class EventLevelExtensions
```

Comment:
nit:  please add an explicit modifier here.   `public` or `internal`.... either one should be fine.